sys/oneway-malloc/oneway-malloc.c

```diff
@@ -20,15 +20,25 @@
 
 #include <string.h>
 
+#include "architecture.h"
 #include "malloc.h"
 
 #define ENABLE_DEBUG 0
 #include "debug.h"
 
+#ifndef ARCHITECTURE_WORD_MASK
+#define ARCHITECTURE_WORD_MASK (ARCHITECTURE_WORD_BYTES - 1)
+#endif
```

Comment:
Is there a use case for configuring this?

Reply:
Not sure why I added this tbh.